### ISSUE TITLE
CELA P1_A11y_code.visualstudio.com_VisualStudioCode_UserandWorkspaceSettings_ScreenReader: Tooltip is not announced.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/vscode-docs#3585]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#CELA P1_A11y_code.visualstudio.com;#A11ySev1;#A11yTCS;#A11yMAS;#Benchmark;#ChromiumEdge;#DesktopWeb;#MAS4.1.2;#MAS1.3.1;#Win10;#Visual Studio Code;#CELAP1_code;#Narrator;#NVDA;#Chrome;

**Environment Details:**
Application Name: code.visualstudio.com
#URL:https://code.visualstudio.com/docs/getstarted/settings
Windows Version: Windows10
Browser version: Microsoft Edge Version 81.0.416.60 (Official build) beta (64-bit), Chrome Version 81.0.4044.113 (Official Build) (64-bit)
Screen reader: narrator ,NVDA2019.3.1

**Repro Steps:**
1)Launch the application url :
code.visualstudio.com/docs/getstarted/settings 
2)User and Workspace Settings Page will get open.
3)Launch screen reader as narrator.
4) Go through heading "Creating User and Workspace Settings" locations)" to the text (Ctrl+Shift+P) using caps+right arrow key.

**Actual:**
On navigating to the text (Ctrl+Shift+P) the tooltip associated with it is not announced.
The control type of the tooltip is set to group.
Similarly for the text " (Ctrl+,)."

**Same with nvda+chrome**

**Same issue in the following page:**
1.code.visualstudio.com/docs/python/python-tutorial 
under headings "Run Hello World/Configure and run the debugger/Install and use packages"
2.https://code.visualstudio.com/docs/getstarted/settings
under heading Language specific editor settings
3.https://code.visualstudio.com/docs/getstarted/settings
under heading Common questions
4.https://code.visualstudio.com/docs/python/python-tutorial
under heading Select a Python interpreter and Create a Python Hello World source code file
5.:https://code.visualstudio.com/docs/editor/versioncontrol
under heading SCMProviders/Branches and Tags/Viewing diffs/Initialize a repository
6. https://code.visualstudio.com/docs/getstarted/tips-and-tricks 
under heading Basics/Status Bar/Customization/Extensions/Files and folders/Editing Hacks/IntelliSense/Git integration/Debugging/Task runner
7.https://code.visualstudio.com/docs/editor/integrated-terminal
under heading Managing multiple terminals/Terminal keybindings

**Expected:**
The tooltip associated with all text should be announced the control of the tooltip should be set to "tooltip" instead of "group"

**User Impact:**
Screen reader users will not receive the n information of the keys that can be used by different os users.

**Attachment link for Reference:**
![image](https://user-images.githubusercontent.com/24580570/79863430-60a6ed00-83f5-11ea-8d9d-b97b23ddeca8.png)
[3585_CELA P1_A11y_code.visualstudio.com_VisualStudioCode_UserandWorkspaceSettings_ScreenReader.zip](https://github.com/microsoft/vscode-docs/files/4515533/3585_CELA.P1_A11y_code.visualstudio.com_VisualStudioCode_UserandWorkspaceSettings_ScreenReader.zip)

**Comment by Tyriar** [comment]
I don't believe this is actually a problem. The tooltip is used to provide advanced information for keybindings that are not the current operating system. Where these would be useful for those using screen readers would be if they use multiple operating systems AND happen to be browsing the VS Code docs on their machine that they don't use VS Code on.

Making this change would degrade the user experience by reading out all platform keybindings when they really are not relevant the vast majority of the time (for any user).

**Comment by sspatil123** [comment]
GithubTags:#Closed;Version 1.75
![image](https://user-images.githubusercontent.com/85863999/217255982-14d95033-6cd9-4bc9-bb3e-a71ce529eb9c.png)
